staging/src/k8s.io/kube-scheduler/config/v1alpha2/types.go

```diff
@@ -20,6 +20,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	componentbaseconfigv1alpha1 "k8s.io/component-base/config/v1alpha1"
+	v1 "k8s.io/kube-scheduler/config/v1"
```

Comment:
Whoa, this is blowing my mind, I would have expected this to be v2alpha1 if you already have a v1?

At any rate, one generally copies types rather than cross-link versions like this.

Reply:
The scheduler has currently 2 APIs: Policy (v1) and componentconfig (v1alpha2)

They aren't necessarily mutually exclusive and once graduated to GA, componentconfig will be appended in v1.

So, is it at that point that we should deduplicate the types? I assume we can still share the internal types, right?

Reply:
ah, I see. This is probably OK then.